src/main/java/redis/clients/jedis/JedisSentineled.java

```diff
@@ -23,6 +24,10 @@ public JedisSentineled(SentineledConnectionProvider sentineledConnectionProvider
     super(sentineledConnectionProvider);
   }
 
+  public JedisSentineled(CommandExecutor executor, SentineledConnectionProvider sentineledConnectionProvider, CommandObjects commandObjects, RedisProtocol redisProtocol) {
```

Comment:
1. We do not plan to change CommandExecutor from here.
```suggestion
  public JedisSentineled(SentineledConnectionProvider sentineledConnectionProvider, CommandObjects commandObjects, RedisProtocol redisProtocol) {
```
2. Add a JavaDoc stating redisProtocol will be set into commandObjects.